src/js/view/view.js

```diff
@@ -905,6 +965,22 @@ function View(_api, _model) {
 
                 addClass(_playerElement, 'jw-flag-floating');
 
+                if (mobileFloatIntoPlace) {
+                    // Creates a dynamic animation where the top of the current player
+                    // Smoothly transitions into the expected floating space in the event
+                    // we can't start floating at 62px
+                    style(_wrapperElement, {
+                        transform: `translateY(-${FLOATING_TOP_OFFSET - playerBounds.top}px)`
```

Comment:
Should this include `transition: null` or `'none'` to clear transitions previously added?

Reply:
those get cleared by the `stopFloating` which should in all cases be called before we hit tthis execution path again